### DISCUSSION TITLE
shuffledns not work with list of domains

### DISCUSSION THREAD

**Discussion by RahmatQurishi** [projectdiscovery/shuffledns#147] [Q&A]
Hi, i want to run shuffledns bruteforce with list of domains.
this command does not work 
cat domains.txt | while read domain; do shuffledns -d $domain -w wordlist.txt -r resolvers.txt -silent;done
is there any switch to do such stuff?!

**Comment by MetzinAround** [comment]
Old questions so I'm closing it, but feel free to re-open with a comment.
